README.md

```diff
@@ -176,6 +176,13 @@ The following exceptions are thrown:
 * `RegistryAuthenticationException`: username and password are invalid
 * `RegistryAuthorizationException`: registry does not support tags using the given credentials, probably because the repository is private and the credentials provided do not have access
 
+#### digest
+````ruby
+manifest = reg.digest("namespace/repo", "2.5.6")
```

Comment:
should be `digest = reg.digest...` since we are getting a digest, not a manifest